src/Application.php

```diff
@@ -156,6 +156,17 @@ public function __call($method, $args)
     }
 
     /**
+     * @param string|Router\Route $path
+     * @param callable|string     $middleware Middleware (or middleware service name) to associate with route.
+     * @param null|string         $name the name of the route
+     * @return Router\Route
+     */
+    public function any($path, $middleware, $name = null)
+    {
+        return $this->route($path, $middleware, $this->httpRouteMethods, $name);
```

Comment:
Pass a `null` or `Router\Route::HTTP_METHOD_ANY` for the third argument. `$httpRouteMethods` only really exists to determine what methods are valid overloading methods.